packages/worker-capacity-manager/package.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "worker-capacity-manager",
+	"version": "1.0.0",
+	"description": "",
+	"main": "index.js",
+	"scripts": {
+		"build": "esbuild --bundle --platform=node --target=node20 --outfile=dist/index.js src/index.ts",
+		"start": "STAGE=DEV nodemon src/index.ts"
+	},
+	"keywords": [],
+	"author": "",
+	"license": "ISC",
+	"dependencies": {
+		"@guardian/transcription-service-common": "1.0.0",
+		"@guardian/transcription-service-backend-common": "1.0.0",
+		"@aws-sdk/client-sqs": "^3.514.0",
+		"@aws-sdk/client-auto-scaling": "^3.514.0"
```

Comment:
client-sqs and client-autoscaling are included in backend-common so don't need to be explicity added here